pkg/tnservice/store_test.go

```diff
@@ -108,7 +108,7 @@ func TestStartWithReplicas(t *testing.T) {
 			return nil, err
 		}
 		return fileservice.NewFileServices(
-			defines.LocalFileServiceName,
+			"",
```

Comment:
Here

Reply:
the first argument of NewFileServices is the default service name. We're removing the default file service.